cmd/percona-dbaas/pxc/pxc.go

```diff
@@ -17,9 +17,16 @@ package pxc
 import (
 	"strings"
 
+	"github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
+var log *logrus.Logger
+
+func init() {
+	log = logrus.New()
+}
+
```

Comment:
No need to define a custom global logger. You already have one in the logrus package.